vapp/src/veCurveVault.vue

```diff
@@ -23,10 +23,13 @@
       button(:disabled='has_allowance_vault', @click.prevent='on_approve_vault') {{ has_allowance_vault ? 'vault approved' : 'approve vault' }}
       button(:disabled='!has_allowance_vault', @click.prevent='on_deposit') deposit {{ crv_balance | fromWei(2) }} CRV
       button(@click.prevent='on_claim') claim {{ claimable | fromWei(2) }} rewards
+    p.row
+      button(:disabled='has_allowance_y3crv_zap', @click.prevent='on_approve_y3crv_zap') {{ has_allowance_y3crv_zap ? 'y3crv zap approved' : 'approve y3crv zap' }}
+      button(:disabled='!has_allowance_y3crv_zap', @click.prevent='on_y3crv_zap') zap {{ claimable | fromWei(2) }} rewards to y3crv vault
     p.row
       button(:disabled='has_allowance_zap', @click.prevent='on_approve_zap') {{ has_allowance_zap ? 'zap approved' : 'approve zap' }}
       button(:disabled='minting_allowed', @click.prevent='on_approve_minter') {{ minting_allowed ? 'minter approved' : 'approve minter' }}
-      button(:disabled='!has_allowance_zap || (need_minter && !minting_allowed)', @click.prevent='on_zap') zap {{ zap_balance | fromWei(2) }} CRV
+      button(:disabled='!has_allowance_zap || (need_minter && !minting_allowed)', @click.prevent='on_zap') zap {{ zap_balance | fromWei(2) }} CRV to Backscratcher
```

Comment:
Maybe it would be better to keep the button text as is, but add a clarification on the side about the zaps.

Reply:
I changed it to look like this:
![image](https://user-images.githubusercontent.com/7820952/98506017-73590600-220f-11eb-934b-4e3b72513168.png)

I wasn't sure how to put the information on the side, so I just put it muted and below. Feel free to adjust this if you'd like